v3/glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+* _channel_ : **TODO: find good description for channel**
+* _cloaking_ : method used to hides telehash traffic on the wire by randomizing all data sent
+* _Cipher Set (CS)_ : a collection of crypto algorithms with a given _cipher set id (CSID)_
+* _Cipher Set ID (CSID)_ : predefined hex number identifying a _cipher set (CS)_
+* _endpoint_ : a participant in the telehash network identified by a **(TODO: or multiple?)** _hashname_
+* _EVEN_ : the _endpoint_ with the numerically lower public key
+* _e3x_ : End-to-End Encrypted eXchange is a flexible encrypted exchange protocol
+* _exchange_ : **TODO: find good description for exchange**
+* _handshake_ : _message_ type used to establish an encrypted _session_ for _channels_
+* _hashname_ : an identifier for a participant of telehash, it is calculated from all public keys of the participants _cipher set (CS)_
+* _LOB_ : Lenght-Oobject-Binary encoding format that allows combining JSON and binary data
+* _link_ : **TODO: is this a term?**
```

Comment:
Yep, I've started using "mesh" and "link" as the definitive /high-level/ concepts for developers, such that they are all you should need to understand to minimally use a telehash library.  A link is just a connection to another endpoint on your mesh, it will have an exchange when it's connected, but a link can exist before you know the public keys if you add it as just a hashname and depend on a router to assist.